en/docs/apis/create-and-publish-your-first-api.md

```diff
@@ -68,15 +62,22 @@ Follow this procedure to publish the API and view the published API on the API D
 
 1. Click **Publish** in the left pane.
 2. On the **Lifecycle Management** page, click **Publish**.
-
-    ![Publish the API](../assets/img/apis/publish.png){.cInlineImage-full}
-
-3. Click **Go to Developer Portal**.
+3. Click **Go to Developer Portal**. Then select an account and proceed to sign in to the API developer portal.
 
     ![Go to API Developer Portal](../assets/img/apis/go-to-devportal.png){.cInlineImage-full}
-
-     You can see the published API in the API Developer Portal.
-
+     
+    You can see the published API in the API Developer Portal.
+    
     ![List of published APIs in the API Developer Portal](../assets/img/apis/devportal.png){.cInlineImage-full}
 
 Congratulations! You have successfully created a REST API and published it on the API Developer Portal.
+
+## Step 5: Invoke the API 
+Follow this procedure to invoke the API from the API Developer Portal:
+
+1. Click **Credentials** in the left pane.
+2. On the **Credentials** page, click **GENERATE ACCESS TOKEN**. Copy the generated access token.
+3. Click **Try Out** in the left pane. 
+4. Paste the copied access token in the **Access Token** field.
+5. Click the `GET` resource you added to expand it.
+6. Click **Try it out** and then click **Execute**. You can see the COVID-19 totals for all US states as the response from the API.
```

Comment:
It seems the page ended abruptly.  Consider rewording and moving the line 73 (`Congratulations!...`) to here.

Reply:
done